code/modules/mob/mob_movement.dm

```diff
@@ -175,6 +175,9 @@
 	if(mob && .)
 		mob.throwing = 0
 
+	for(var/obj/O in mob)
+		O.on_mob_move(direct, src)
```

Comment:
Can we really justify this? Having each of your clothes call a proc every single time we move? Just to add a jetpack thing?